config/crd/bases/anywhere.eks.amazonaws.com_clusters.yaml

```diff
@@ -541,6 +547,12 @@ spec:
                 description: Descriptive message about a fatal problem while reconciling
                   a cluster
                 type: string
+              reconciledGeneration:
```

Comment:
what do you think about adding here in the description that these are for internal use and there is no grantee around them. same in both fields

Reply:
Good callout. will add a note